### PR TITLE
defer reader close

### PR DESCRIPTION
**Pull request by pweil-**
Fixes https://github.com/openshift/image-inspector/issues/4

@simon3z @enoodle

### CODE REVIEW

**Comment by simon3z** [comment]
LGTM :+1: @enoodle?

File: pkg/inspector/image-inspector.go [hunk 1]
```diff
@@ -336,6 +336,7 @@ func appendDockerCfgConfigs(dockercfg string, cfgs *docker.AuthConfigurations) e
 	if err != nil {
 		return fmt.Errorf("Unable to open docker config file: %v\n", err)
```
Comment (enoodle) [inline pkg/inspector/image-inspector.go]: @pweil- we are still missing this ```return``` what about after line 335?

Comment (pweil-) [inline pkg/inspector/image-inspector.go]: @enoodle if the error is not nil then the reader will be nil so it's ok to skip this return

Comment (enoodle) [inline pkg/inspector/image-inspector.go]: Ok

**Comment by enoodle** [comment]
LGTM :+1:

**Pull request merged**